website/docs/d/kubernetes_cluster.html.markdown

```diff
@@ -256,6 +258,18 @@ A `azure_policy` block supports the following:
 
 * `enabled` - Is Azure Policy for Kubernetes enabled?
 
+An `ingress_application_gateway` block supports the following:
+
+* `enabled` -  Has this cluster been integrated with an application gateway.
```

Comment:
Can we modify the wording a bit here to be consistent with the one in the resource document?
```suggestion
* `enabled` -  Is the Application Gateway ingress controller integrated with this Kubernetes Cluster?
```